src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java

```diff
@@ -216,24 +223,39 @@ private void removeSessionLevelPrivileges(InstructorAttributes instructorToEdit,
         return sectionNamesMap;
     }
 
-    private void updateInstructorPrivilegesForSectionInSectionLevel(String sectionParam, List<String> sectionNames, InstructorAttributes instructorToEdit) {
-        boolean isViewStudentInSectionsChecked = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS + sectionParam) != null;
-        boolean isViewCommentInSectionsChecked = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_COMMENT_IN_SECTIONS + sectionParam) != null;
-        boolean isGiveCommentInSectionsChecked = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS + sectionParam) != null;
-        boolean isModifyCommentInSectionsChecked = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COMMENT_IN_SECTIONS + sectionParam) != null;
+    private void updateInstructorPrivilegesForSectionInSectionLevel(
+            String sectionParam, List<String> sectionNames, InstructorAttributes instructorToEdit) {
```

Comment:
Maybe leave `String sectionParam` above, and indent second line 4 spaces from method name, like this from the style guide:
![image](https://cloud.githubusercontent.com/assets/14101781/15662582/97c345c2-2724-11e6-8830-081c1e825fd5.png)